PublicMapping/districtbuilder-classic
# Fix statistics pane in plan view

## Overview

`humanize` wasn't in our installed apps, which caused a template error when
trying to render the statistics pane. This PR adds `django.contrib.humanize` to
our installed apps.

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/5702984/35929403-6f757300-0bfd-11e8-8f99-8464a696296f.png)

### Notes

No I'm not kidding, but yes I wish I were

## Testing Instructions

 * bring up your server
 * view a plan
 * check out the stats
